DQM/EcalMonitorTasks/src/RawDataTask.cc

```diff
@@ -61,7 +61,7 @@ namespace ecaldqm {
     // Get GT L1 info
     const FEDRawData &gtFED(_fedRaw.FEDData(812));
     if(gtFED.size() > sizeof(uint64_t)){ // FED header is one 64 bit word
-      uint32_t *halfHeader((uint32_t *)gtFED.data());
+      const uint32_t *halfHeader = reinterpret_cast<const uint32_t *>(gtFED.data());
```

Comment:
@makortel for my understanding, could you please explain why here reinterpret_cast is preferable to static_cast?

Reply:
@fabiocos 
The return type of `FEDRawData::data()` is `const unsigned char *`, which then gets re-interpreted here as `const uint32_t *`.

Reply:
ok, there is no relationship among the original and casted version...